board_firmware/firmware/src/state/runtime/AInRuntimeConfig.h

```diff
@@ -77,11 +77,11 @@ extern "C" {
     }AInCalParam;
 
     // Define a storage class for analog input modules
-    #define MAX_AIN_RUNTIME_MOD 3
```

Comment:
I wonder if we even need runtime MAX_AIN_RUNTIME_MOD or MAX_AIN_RUNTIME_CHANNEL when we have MAX_AIN_MOD and MAX_AIN_CHANNEL.  I'm not sure those would ever be different.

Reply:
Yes, many defines might not be needed, but i will arrange them slowly. But for NQ1 no AI module is one, right?